src/deploy-cromwell-on-azure/scripts/startup.sh

```diff
@@ -90,28 +89,41 @@ done
 write_log "Account access OK"
 write_log
 
+write_log "Mounting containers (default storage account = $storage_account_name)"
+./mount_containers.sh -a $storage_account_name
+write_log
+
+write_log "Mounted containers:"
+findmnt -t fuse
+write_log
+
+k=docker-compose.*.yml
```

Comment:
Maybe easier to read:
```
compose_files=(docker-compose.*.yml)
compose_files=("docker-compose.yml" "${compose_files[@]}")
kv["COMPOSE_FILE"]=$(IFS=:; echo "${compose_files[*]}")

```

Reply:
I think there's an error in your code, bash isn't my strong suit so I can't easily debug it but I think it's breaking it up weird. 
```
Pulling tes            ... error
Pulling cromwell       ... error
Pulling triggerservice ... error

ERROR: for tes  Get "http:": http: no Host in request URL

ERROR: for triggerservice  Get "http:": http: no Host in request URL

ERROR: for cromwell  Get "http:": http: no Host in request URL
Get "http:": http: no Host in request URL
Get "http:": http: no Host in request URL
Get "http:": http: no Host in request URL
```

Reply:
"no Host in request URL" error is related to docker pull, and I don't see a connection. It worked for me.

Reply:
idk, i replaced the code with the previous version and it worked. Given this needs further debugging I'm gonna mark as "Wont Fix" for now and it can be refactored in the future.